src/main/webapp/gameStage.html

```diff
@@ -13,7 +13,7 @@
   <script src="gameDialogue.js"></script>
 </head>
 <body onload = "loadFunctions()">
-  <h2 class="step-outline">Welcome!</h2>
+  <h2 class="step-outline">Welcome! Player, please press the next button to begin todays lecture</h2>
```

Comment:
Now that the lecture already begins onload, this might be a little inaccurate/unneeded?

Reply:
+1, I would remove this text